docs/source/llms/transformers/guide/index.rst

```diff
@@ -648,4 +678,156 @@ This input format requires that both the bitrate has been set prior to conversio
+
+In MLflow 2.11.0, we introduced support for PEFT models in the Transformers flavor. You can log and load PEFT models using the same APIs as other Transformers models, such as :py:func:`mlflow.transformers.log_model()` and :py:func:`mlflow.transformers.load_model()`.
+
+.. code-block:: python
+
+    import mlflow
+    from peft import LoraConfig, get_peft_model
+    from transformers import AutoModelForCausalLM, AutoTokenizer
+
+    model_id = "databricks/dolly-v2-7b"
+    base_model = AutoModelForCausalLM.from_pretrained(model_id)
+    tokenizer = AutoTokenizer.from_pretrained(model_id)
+
+    peft_config = LoraConfig(...)
+    peft_model = get_peft_model(base_model, peft_config)
+
+    with mlflow.start_run():
+        # Your training code here
+        ...
+
+        # Log the PEFT model
+        model_info = mlflow.transformers.log_model(
+            transformers_model={
+                "model": peft_model,
+                "tokenizer": tokenizer,
+            },
+            artifact_path="peft_model",
+        )
+
+    # Load the PEFT model
+    loaded_model = mlflow.transformers.load_model(model_info.model_uri)
+
+PEFT Models in MLflow Tutorial
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Check out the tutorial `Fine-Tuning Open-Source LLM using QLoRA with MLflow and PEFT <../tutorials/transformers_peft.ipynb>`_ for a more in-depth guide on how to use PEFT with MLflow,
+
+Format of Saved PEFT Model
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+When saving PEFT models, MLflow only saves the PEFT adapter and the configuration, but not the base model weight. This is the same behavior as the Transformer's `save_pretrained() <https://huggingface.co/docs/transformers/v4.38.1/en/main_classes/model#transformers.PreTrainedModel.save_pretrained>`_ method and is super efficient in terms of storage space and logging latency. One difference is that MLflow alsot saves the HuggingFace Hub repository name and version for the base model in the model metadata, so that it can load the same base model when loading the PEFT model. Concretely, the following artifacts are saved in MLflow for PEFT models:
+
+* The PEFT adapter weight under the ``/peft`` directory.
+* The PEFT configuration as a JSON file under the ``/peft`` directory.
+* The HuggingFace Hub repository name and commit hash for the base model in the ``MLModel`` metadata file.
+
+Limitations of PEFT Models in MLflow
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Since the model saving/loading behavior for PEFT models is similar to that with ``save_pretrained=False``, :ref:`the same caveats <caveats-of-save-pretrained>` apply to PEFT models. For example, the base model weight may be deleted or become private in the HuggingFace Hub, and PEFT models cannot be registered to the legacy Databricks Workspace Model Registry.
```

Comment:
```suggestion
Since the model saving/loading behavior for PEFT models is similar to that of ``save_pretrained=False``, :ref:`the same caveats <caveats-of-save-pretrained>` apply to PEFT models. For example, the base model weight may be deleted or become private in the HuggingFace Hub, and PEFT models cannot be registered to the legacy Databricks Workspace Model Registry.
```